### ISSUE TITLE
cordova-plugin-crosswalk-webview-v3 (cordova 9 support and bug fixes)

### ISSUE THREAD

**Issue by ardabeyazoglu** [crosswalk-project/cordova-plugin-crosswalk-webview#263]
Since the repo is not maintained anymore, no pull request is merged and simple problems take too much time to solve for multiple people. So, i created an updated plugin from my fork that supports latest cordova. Anybody who wants to contribute and help maintaining for small bug fixes are welcome.

Here: https://github.com/ardabeyazoglu/cordova-plugin-crosswalk-webview-v3

**Comment by SuperStar518** [comment]
You did really a good job!

**Comment by gonzalovinas** [comment]
Great! - BTW... is crosswalk still needed for current Android > 7 devices ?

**Comment by ardabeyazoglu** [comment]
@gonzalovinas No, android 7+ has higher chromium version than crosswalk. But for 6 and lower, crosswalk has higher chromium.
